internal/evalhub_mcp/server/server_test.go

```
@@ -3,13 +3,253 @@ package server
+	srv := New(info, discardLogger)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	serverTransport, clientTransport := mcp.NewInMemoryTransports()
+
+	serverSession, err := srv.Connect(ctx, serverTransport, nil)
+	if err != nil {
+		t.Fatalf("server.Connect failed: %v", err)
+	}
+	defer serverSession.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `serverSession.Close` is not checked

(errcheck)